File: app/src/main/java/com/simulando/joaopaulodribeiro/simulando/model/student/AuthStudentResponse.java
package com.simulando.joaopaulodribeiro.simulando.model.student;

/**
 * Created by joao.paulo.d.ribeiro on 23/09/2017.
 */

public class AuthStudentResponse {

    String status;
    Data data;
    String token;
    Long expire_at;

    public String getStatus() {
        return status;
    }

    public Data getData() {
        return data;
    }

    public String getToken() {
        return token;
    }

    public Long getExpire_at() {
        return expire_at;
    }

    /******************************************/
    public class Data {
        Integer id;
        String email;
        String password;
        String role;
        String name;
        Integer school_id;
        Student student;

        public Integer getId() {
            return id;
        }

        public String getName() {
            return name;
        }

        public Integer getSchool_id() {
            return school_id;
        }

        public Student getStudent() {
            return student;
        }
    }
}

File: app/src/main/java/com/simulando/joaopaulodribeiro/simulando/model/student/AuthStudentResponse.kt
package com.simulando.joaopaulodribeiro.simulando.model.student

/**
 * Created by joao.paulo.d.ribeiro on 23/09/2017.
 */

class AuthStudentResponse {

    var status: String = ""
    var data: Data? = null
    var token: String = ""
    var expire_at: Long = -1

    inner class Data {
        var id: Int = -1
        var email: String = ""
        var password: String = ""
        var role: String = ""
        var name: String = ""
        var school_id: Int = -1
        var student: Student? = null
    }
}
